addons/l10n_nl/migrations/3.4/post-migrate.py

```diff
@@ -0,0 +1,7 @@
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    for company in env['res.company'].search([('chart_template', '=', 'nl')]):
```

Comment:
To follow a fix made recently on all the similar scripts
```suggestion
    for company in env['res.company'].search([('chart_template', '=', 'nl')], order="parent_path"):
```